src/main/java/org/scijava/log/DefaultLogger.java

```diff
@@ -31,10 +31,8 @@
 
 package org.scijava.log;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Properties;
+import java.util.*;
```

Comment:
SciJava style is to never use wildcard imports.

And in general: please keep changes in a patch set atomic, and matching what the commit message says is being changed. If you really wanted to change the imports like this, it should be its own commit.